src/kernel/bootstrap/Globals.rb

```diff
@@ -158,12 +158,12 @@
 # RUBY_PLATFORM  is installed in RubyContext>>_initTransient:
 RUBY_VERSION = '1.9.3'
 # Note - the packager modifies the date on any line starting with RUBY_RELEASE_DATE
-RUBY_RELEASE_DATE = '2013-05-17'
+RUBY_RELEASE_DATE = '2015-07-07'
 RUBY_PATCHLEVEL = 327
 RUBY_DESCRIPTION = 'ruby 1.9.3 (maglev patchlevel 327)'
 RUBY_COPYRIGHT = 'ruby - Copyright (C) 1993-2010 Yukihiro Matsumoto; maglev additions Copyright(C) 2009-2010 GemStone Systems Inc.'
 
-VERSION = '1.2Alpha4'
+VERSION = '1.2Alpha6'
```

Comment:
The fact that this version was skewed from version.txt probably means that one should use the other...